Core/include/Acts/EventData/MultiTrajectory.hpp

```diff
@@ -432,18 +432,18 @@ class MultiTrajectory {
   }
 
   /// Retrieve a parameter proxy instance for parameters at a given index
-  /// @param parIdx Index into the parameter column
+  /// @param istate Index into the parameter column
   /// @return Mutable proxy
   template <bool RO = ReadOnly, typename = std::enable_if_t<!RO>>
-  typename TrackStateProxy::Parameters parameters(IndexType parIdx) {
-    return self().parameters_impl(parIdx);
```

Comment:
I think this was part of another PR as well, an I think the renaming is incorrect.